examples/qeq/run_all.py

```
@@ -0,0 +1,170 @@
+    -------
+    bool
+        True if successful, False otherwise
+    """
+    print(f"\n{'='*80}")
+    print(f"Running: {script_name}")
+    print(f"Description: {description}")
+    print(f"{'='*80}")
+
+    try:
+        start_time = time.time()
+        result = subprocess.run(
```

Ruff (0.14.14)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)